doc/api/process.markdown

```diff
@@ -7,14 +7,21 @@ It is an instance of [`EventEmitter`][].
 
 ## Event: 'beforeExit'
 
-This event is emitted when Node.js empties its event loop and has nothing else to
-schedule. Normally, Node.js exits when there is no work scheduled, but a listener
-for `'beforeExit'` can make asynchronous calls, and cause Node.js to continue.
+This event is emitted when Node.js empties its event loop and has nothing else
+to schedule. Normally, Node.js exits when there is no work scheduled, but a
+listener for `'beforeExit'` can make asynchronous calls, and cause Node.js to
+continue.
 
-`'beforeExit'` is not emitted for conditions causing explicit termination, such as
-[`process.exit()`][] or uncaught exceptions, and should not be used as an
+`'beforeExit'` is not emitted for conditions causing explicit termination, such
+as [`process.exit()`][] or uncaught exceptions, and should not be used as an
 alternative to the `'exit'` event unless the intention is to schedule more work.
 
+## Event: 'disconnect'
+
+If Node.js was spawned with an IPC channel, this event will be emitted when
+either [`process.disconnect()`][] or [`ChildProcess.disconnect()`] in parent
```

Comment:
this is unclear, it could be read as `(process.disconnect or ChildProcess.disconnect) in parent`, or as `process.disconnect or (ChildProcess.disconnect in parent)`. the latter is correct, but I misread as the former. maybe add `in the child process` after process.disconnect()